src/Support/Http/Controllers/GraphQLController.php

```diff
@@ -32,11 +34,14 @@ public function __invoke(
         $result = $graphQL->executeOperationOrOperations($operationOrOperations, $context);
 
         $response = $createsResponse->createResponse($result);
+        $response = $response->setCache($cacheControl->makeHeaderOptions());
 
         $eventsDispatcher->dispatch(
             new EndRequest($response)
         );
 
+        app()->forgetInstance(CacheControl::class);
```

Comment:
Can we do this in a listener of `EndRequest`? That would mean that users who exclude `CacheControlServiceProvider` don't pay for it.